doc/readthedocs/geometrydefinition.rst

```diff
@@ -181,16 +180,23 @@ For SPH, the following different phases may be defined:
 - neumannphase
 - dirichletphase
 
-The radius of the particles are by default defined in the material definition. For SPH, this is the only option available.
-For DEM, however, as mentioned above, they can also be defined per particle.
-For this definition option, one has to define the option :ref:`INITIAL_RADIUS <particledynamic_dem_initial_radius>` in :ref:`PARTICLE DYNAMIC <SECparticledynamic_dem>`::
+Note that with SPH the particles are allowed to change its phase during a simulation based on the input parameter ``PHASECHANGETYPE`` and ``PHASECHANGEDEFINITION`` in the section :ref:`PARTICLE DYNAMIC/SPH <SECparticledynamic_sph>`.
+
+For SPH the radius corresponds to the cutoff radius of the smoothing kernel. Since the cutoff radius is assumed to be constant this is the only option available.
+
+For the DEM, the particles are assumed to have a spherical shape with a given radius.
+
+By default, the radius of the particles are defined in the material definition for both DEM and SPH.
+
+However, as mentioned above, with DEM the radius can also be defined individually per particle.
+To do so one has to define the option :ref:`INITIAL_RADIUS <particledynamic_dem_initial_radius>` in :ref:`PARTICLE DYNAMIC/DEMi <SECparticledynamic_dem>`::
 
     ------------------PARTICLE DYNAMIC/DEM
     INITIAL_RADIUS    RadiusFromParticleInput
 
-Additionally, it is possible to define a normal or a log-normal distribution, defined by two parameters, :math:`\mu` and :math:`\sigma`.
-The parameter :math:`mu` is defined by the initial radius, i.e., in the material definition,
-while the parameter :math:`\sigma` is defined by the parameter :ref:`RADIUSDISTRIBUTION_SIGMA <particledynamic_dem_radiusdistribution_sigma>` in :ref:`PARTICLE DYNAMIC <SECparticledynamic_dem>`::
+Additionally, it is possible to define a normal or a log-normal distribution of the radius, defined by two parameters, :math:`\mu` and :math:`\sigma`.
```

Comment:
Actually, what does a size distribution mean? Is the radius chosen randomly within the distribution? I am not a particle expert, but maybe it is worth including a more in-depth explanation.

Reply:
Typically in powder spreading applications the individual powder grains are not of equal size, i.e., with given fixed radius. Accordingly, based on a characterization of the powder the size of particles can be described by a distribution (normal or log-normal) with mean and standard deviation. @ppraegla can probably give you more information or also link one of the papers from our group. In the implementation, the radius of each particle is at the beginning of a simulation randomly set based on the selected distribution.